### PR TITLE
Fix display rollup information incorrect

### LINKED ISSUE #1620: Display ROLLUP and MATERIALIZED VIEW is incorrect
Show rollup information is inconsistent with schema.
### Steps to reproduce the behavior (Required)
show alter table rollup;
SHOW ALTER TABLE MATERIALIZED VIEW ;
### Expected behavior (Required)
Correct information
### Real behavior (Required)
image

### StarRocks version (Required)
 main

### PR DESCRIPTION
**Pull request by stephen-shelby**
The information is not right when using `show alter table rollup`. The value is not corresponds to the tab bar.
The pull request adjust the position to make it be right.

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/analysis/ShowAlterStmt.java [hunk 1]
```diff
@@ -254,9 +254,9 @@ public ShowResultSetMetaData getMetaData() {
         ShowResultSetMetaData.Builder builder = ShowResultSetMetaData.builder();
 
         ImmutableList<String> titleNames = null;
-        if (type == AlterType.ROLLUP) {
+        if (type == AlterType.ROLLUP || type == AlterType.MATERIALIZED_VIEW) {
             titleNames = RollupProcDir.TITLE_NAMES;
-        } else if (type == AlterType.COLUMN || type == AlterType.MATERIALIZED_VIEW) {
+        } else if (type == AlterType.COLUMN ) {
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/analysis/ShowAlterStmt.java:259]:
```suggestion
        } else if (type == AlterType.COLUMN) {
```

Comment (stephen-shelby) [reply fe/fe-core/src/main/java/com/starrocks/analysis/ShowAlterStmt.java]: done

File: fe/fe-core/src/main/java/com/starrocks/alter/RollupJobV2.java [hunk 2]
```diff
@@ -708,7 +708,6 @@ protected void getInfo(List<List<Comparable>> infos) {
         info.add(baseIndexName);
         info.add(rollupIndexName);
         info.add(rollupIndexId);
-        info.add(Integer.toString(rollupSchemaHash));
```
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/alter/RollupJobV2.java:711]: daily test will failed because of the position changed.

**Review by chaoyli — APPROVED** [review]

**Review by imay — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/alter/RollupJobV2.java (hunk 2 above)
Comment (stephen-shelby) [reply fe/fe-core/src/main/java/com/starrocks/alter/RollupJobV2.java:711]: daily test case has been updated

**Pull request merged**